crates/ty_python_semantic/resources/primer/bad.txt

```diff
@@ -1,24 +1,21 @@
 Tanjun  # hangs
-altair  # cycle panics (try_metaclass_)
 antidote  # hangs / slow
 artigraph  # cycle panics (value_type_)
-colour  # cycle panics (try_metaclass_)
 core  # cycle panics (value_type_)
 cpython  # access to field whilst being initialized, too many cycle iterations
 discord.py  # some kind of hang, only when multi-threaded?
-freqtrade  # cycle panics (try_metaclass_)
-hydpy  # cycle panics (try_metaclass_)
-ibis  # cycle panics (try_metaclass_)
+freqtrade  # hangs
+hydpy  # too many iterations
 manticore  # stack overflow, see https://github.com/astral-sh/ruff/issues/17863
 pandas  # slow
-pandas-stubs  # cycle panics (try_metaclass_)
-pandera  # cycle panics (try_metaclass_)
+pandas-stubs  # salsa cycle.rs:164 assertion fails
```

Comment:
😨

Reply:
```suggestion
pandas-stubs  # salsa cycle.rs assertion fails (https://github.com/salsa-rs/salsa/issues/831)
```

Reply:
Does the assertion consistently reproduce when checking `pandas-stubs`? I just saw it occur in a mypy_primer run on https://github.com/astral-sh/ruff/pull/17897 (<https://github.com/astral-sh/ruff/actions/runs/14880713337/job/41787973995>) but it went away when I reran the job.

Reply:
Not consistently -- it either seems to hang, or it hits that assertion and then finishes.